Packaging.props

```diff
@@ -1,5 +1,6 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <LicenseUrl>https://github.com/dotnet/corefx/blob/master/LICENSE</LicenseUrl>
```

Comment:
This is the LicenseUrl that will go into our nuget packages and we are just linking back the the corefx MIT license for these packages. @richlander @terrajobst